config/shared/errors/errors.go

```diff
@@ -91,6 +89,10 @@ var (
 
 	// AWS S3 specific errors
 	ErrInvalidS3ObjectVersionId = errors.New("invalid S3 object VersionId")
+
+	// Obsolete errors, left here for ABI compatibility
+	ErrFilePermissionsUnset      = errors.New("permissions unset, defaulting to 0644")
+	ErrDirectoryPermissionsUnset = errors.New("permissions unset, defaulting to 0755")
```

Comment:
While at it, would it be worth updating the label to `-rwxr-xr-x`, as the above is actually not a good integer for Ignition/JSON? (Even if I think this won't be visible anymore)

Reply:
I don't think so?  I thought about replacing the strings with "error no longer used" or similar, but opted to leave the historical text as documentation.